runtime/src/accounts_db.rs

```diff
@@ -1410,12 +1409,23 @@ impl AccountsDB {
         let store =
             Arc::new(self.new_storage_entry(slot, &Path::new(&self.paths[path_index]), size));
         let store_for_index = store.clone();
-        let slot_storage = self
+
+        let slot_storages: SlotStores = self
```

Comment:
`: SlotStore  = `: i like the taste of well nuanced explicit type annotation at the age of rampant type inference xD